src/components/SearchBar/SearchBar.mdx

```diff
@@ -9,10 +9,10 @@ import Link from "../Link/Link";
 
 # SearchBar
 
-| Component Version | DS Version |
-| ----------------- | ---------- |
-| Added             | `0.0.4`    |
-| Latest            | `3.6.1`    |
+| Component Version | DS Version   |
```

Comment:
Since you're already updating this doc page.... can you include the new layout styles? Use the `ComponentDocsHeader` component and check `Banner` (https://nypl-design-system-git-reservoir-v4-nypl.vercel.app/?path=/docs/components-feedback-banner--docs) for an example of the new order for storybook doc pages.